_code-samples/get-started/js/get-acct-info.js

```diff
@@ -1,39 +1,63 @@
 // Import the library
-const xrpl = require("xrpl")
+// @chunk {"steps": ["connect-tag"]}
+import xrpl from "xrpl"
 
-// Wrap code in an async function so we can use await
-async function main() {
+// Define the network client
+const SERVER_URL = "wss://s.altnet.rippletest.net:51233/"
+const client = new xrpl.Client(SERVER_URL)
+await client.connect()
+console.log("Connected to Testnet")
+// @chunk-end
 
-  // Define the network client
-  const SERVER_URL = "wss://s.altnet.rippletest.net:51233/"
-  const client = new xrpl.Client(SERVER_URL)
-  await client.connect()
+// @chunk {"steps": ["get-account-create-wallet-tag"]}
+// Create a wallet and fund it with the Testnet faucet:
+console.log("\nCreating a new wallet and funding it with Testnet XRP...")
+const fund_result = await client.fundWallet()
+const test_wallet = fund_result.wallet
+console.log(`Wallet: ${test_wallet.address}`)
+console.log(`Balance: ${fund_result.balance}`)
+console.log(`\nAccount Testnet Explorer URL:\nhttps://testnet.xrpl.org/accounts/${test_wallet.address}`)
```

Comment:
The fact that this line is long (to col 104) matters more when displaying the code samples side by side, so it's better to break it up:

```suggestion
console.log('Account Testnet Explorer URL:')
console.log(`  https://testnet.xrpl.org/accounts/${test_wallet.address}`)
```